includes/MigrateStore.php

```
@@ -156,37 +159,109 @@ public function handle_export_action()
 		$uploaded_file    = wp_handle_upload($_FILES['json_zip_file'], $upload_overrides);
 
 		if (! $uploaded_file || isset($uploaded_file['error'])) {
-			wp_die($uploaded_file['error'] ?? 'File upload failed');
+			wp_die( esc_html( $uploaded_file['error'] ?? 'File upload failed' ) );
 		}
 
-		$uploaded_file_name     = sanitize_file_name($_FILES['json_zip_file']['name']);
+		$uploaded_file_name     = isset( $_FILES['json_zip_file']['name'] )
+			? sanitize_file_name( wp_unslash( $_FILES['json_zip_file']['name'] ) )
+			: '';
 		$uploaded_file_basename = basename($uploaded_file_name, '.zip');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$uploaded_file_basename'. (undefined)

(UnusedLocalVariable)